get-started/mcp/self-hosted/tools.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Tools Reference
+description: Technical reference for the Verify service tools in kluster.ai's example MCP server implementation.
+---
+
+# Tools Reference
+
+These tools provide access to the [Verify service](/get-started/verify/reliability/overview){target=self} for reliability checking and document verification.
+
+## Tool Overview
+
+| Tool | Purpose | Best For |
+|:---|:---|:---|
+| `verify` | Verify standalone claims. | General statements, trivia, current events, news. |
+| `verify_document` | Verify claims about documents. | Quotes, data extraction, RAG hallucination checking. |
+
+### Verify
+
+Verifies any statement against reliable online sources.
+
+#### Parameters
```

Comment:
We avoid using `####` - so maybe thre is a different way to present this?

Reply:
I really like that idea @eshaben - maybe let's track this with a ticket to get back to this? Or is something it can be implemented quickly